initgeneration/init.sh.tmpl

```diff
@@ -116,6 +116,25 @@ tenant ${host_tenant}
 isCloudNativeFullStack true" >> ${1}
 }
 
+writeConfEnrichmentTo() {
+  mkdir -p "${1}"
+  echo "k8s.pod.uid ${K8S_PODUID}
+k8s.pod.name ${K8S_PODNAME}
+k8s.namespace.name ${K8S_NAMESPACE}
+dt.kubernetes.workload.kind ${DT_WORKLOAD_KIND}
+dt.kubernetes.workload.name ${DT_WORKLOAD_NAME}
+dt.kubernetes.cluster.id ${cluster_id}" >> "${1}/${2}.properties"
+
+  echo "{
+\"k8s.pod.uid\": \"${K8S_PODUID}\",
+\"k8s.pod.name\": \"${K8S_PODNAME}\",
+\"k8s.namespace.name\": \"${K8S_NAMESPACE}\",
+\"dt.kubernetes.workload.kind\": \"${DT_WORKLOAD_KIND}\",
+\"dt.kubernetes.workload.name\": \"${DT_WORKLOAD_NAME}\",
+\"dt.kubernetes.cluster.id\": \"${cluster_id}\"
+}" >> "${1}/${2}.json"
```

Comment:
Appending to a properties file is okay. Appending to a JSON file is not. Is someone else allowed to contribute to these files? I propose to have clear ownership and therefore overwrite them.

Reply:
First of all dt_metadata.properties and dt_metadata.json files must be consistent with each other so both files are overwritten.